### ISSUE TITLE
Inconsistency in prefixing of metadata with `radar_` and `lidar_`

### ISSUE THREAD

**Issue by marack** [NCAR/CfRadial#5]
We have several optional groups that are specified at the global level as a way to cater for radar or lidar specific metadata.  These are `radar_parameters`, `lidar_parameters`, `radar_calibration`, and `lidar_calibration`.

All of the variables specified under `radar_parameters` are prefixed with `radar_`.  For example `radar_antenna_gain`.  However under `radar_calibration` we do not prefix variable names, for example `receiver_gain_hc`.

Since the variables stored within these groups are already known to relate to radar or lidar by virtual of the group name, the use of a `radar_` prefix is redundant.

Can we remove the `radar_` and `lidar_` prefixes from the variable names in the two parameter tables (sections `7.1` and `7.2`)?

**Comment by mike-dixon** [comment]
Agreed - we should remove radar_ and lidar_ prefixes from the elements within the radar_parameters and lidar_parameters groups.

While we are busy with that, should we revisit the monitoring group? We have some radar_specific parameters in there. We could have a radar_monitoring and lidar_monitoring group instead.
